### PR TITLE
[0.13.x] KOGITO-2921  Bump to Quarkus 1.7.0.CR1

### PR DESCRIPTION
**Pull request by evacchi**
- https://github.com/kiegroup/kogito-runtimes/pull/671
- https://github.com/kiegroup/kogito-apps/pull/367
- https://github.com/kiegroup/kogito-examples/pull/318

### CODE REVIEW

**Comment by evacchi** [comment]
Jenkins please retest this

**Comment by evacchi** [comment]
so infinispan-related failures do not surprise me https://github.com/kiegroup/kogito-runtimes/pull/672 -- the others, a little bit... :thinking:

**Comment by evacchi** [comment]
I am trying to fold https://github.com/kiegroup/kogito-runtimes/pull/672 into this one I think they can't really stand alone

File: pom.xml [hunk 1]
```diff
@@ -117,18 +117,18 @@
...
     <version.org.antlr>3.5.2</version.org.antlr>
     <version.org.antlr4>4.8</version.org.antlr4>
     <version.org.assertj>3.16.1</version.org.assertj>
     <version.org.eclipse.jdt>3.19.0</version.org.eclipse.jdt>
     <version.org.hamcrest>1.3</version.org.hamcrest> <!-- else old version coming from Mockito wins and breaks tests -->
-    <version.org.infinispan>10.1.5.Final</version.org.infinispan>
-    <version.org.infinispan.image>10.1.5.Final</version.org.infinispan.image>
+    <version.org.infinispan>11.0.1.Final</version.org.infinispan>
+    <version.org.infinispan.image>11.0.1.Final</version.org.infinispan.image>
     <version.org.infinispan.protostream>4.3.2.Final</version.org.infinispan.protostream>
```
Comment (cristianonicolai) [inline pom.xml:132]: protostream is now 4.3.3.Final

File: pom.xml [hunk 2]
```diff
@@ -663,11 +662,6 @@
         <version>${version.org.graalvm.nativeimage}</version>
       </dependency>
 
-      <dependency>
-        <groupId>io.smallrye.reactive</groupId>
-        <artifactId>smallrye-reactive-messaging-provider</artifactId>
```
Comment (cristianonicolai) [inline pom.xml:668]: can we keep this in the 0.13.x branch shouldnt prevent the upgrade, right?

Comment (evacchi) [reply pom.xml:668]: it was conflicting at places

**Comment by evacchi** [comment]
jenkins please retest this

**Comment by evacchi** [comment]
jenkins please retest this

File: pom.xml (hunk 2 above)
Comment (evacchi) [reply pom.xml:668]: let me do another green build _without_ this, then maybe I'll try another run with this back in

**Comment by evacchi** [comment]
wheel of fortune

jenkins please retest this

**Pull request merged**